app/components/site-chrome/template.hbs

```diff
@@ -109,23 +109,23 @@
           {{/if}}
         </li>
 
-        {{#link-to 'stream' tagName='li' href=false class="list-item list-item--nav" }}
+        {{#link-to 'stream' tagName='li' href=false class="list-item list-item--nav"}}
           {{link-to 'All Streams' 'stream' class="inherit" preventDefault=false}}
         {{/link-to}}
 
-        {{#link-to 'schedule' current-when="schedule.date" tagName='li' href="false" class="list-item list-item--nav" }}
+        {{#link-to 'schedule' current-when="schedule.date" tagName='li' href=false class="list-item list-item--nav"}}
```

Comment:
there's no rule, but I prefer single quotes in all cases and escaping apostrophes in longer strings, but that's me.